src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp

```diff
@@ -2119,7 +2168,7 @@ FixedwingPositionControl::tecs_update_pitch_throttle(const hrt_abstime &now, flo
 		float pitch_min_rad, float pitch_max_rad,
 		float throttle_min, float throttle_max, float throttle_cruise,
 		bool climbout_mode, float climbout_pitch_min_rad,
-		uint8_t mode, float hgt_rate_sp)
+		bool disable_underspeed_detection, float hgt_rate_sp)
```

Comment:
Why do we want to remove TECS_MODE setting as a vairable?

Reply:
@Jaeyoung-Lim Because it's just used at one place to figure out if underspeed detection should be disabled. It's cleaner to just pass that flag directly.

Reply:
Yes, and because the TECS_MODE currently (even before this PR) is set by TECS, and not passed into TECS as an argument. Not saying that I like this a lot..